### DISCUSSION TITLE
DietPi-Software | ccxt cryptocurrency trading API

### DISCUSSION THREAD

**Discussion by faustf** [MichaIng/DietPi#6110] [Software Request]
#### Formal software information
- Software title | ccxt
- Short description | lybrery for multiconnect with many crypto exchange 
- Official URL (if available) | https://github.com/ccxt/ccxt
- GitHub URL (if available) | https://github.com/ccxt/ccxt
- (Official) install documentation | https://docs.ccxt.com/en/latest/install.html
- 
#### Are there similar/alternative software titles available with DietPi-Software?
not exist 
- ...

#### What makes your requested software better than the above solutions, if available?
A JavaScript / Python / PHP cryptocurrency trading API with support for more than 100 bitcoin/altcoin exchanges 
- ...

#### How can DietPi make the installation easier or compatible, than following the install instructions or do APT installation, if available?
i dont know 
- ...

#### Can you provide the installation steps that you would suggest DietPi-Software to do?

- ...

**Comment by MichaIng** [comment]
Many thanks for your request.

I converted it into a discussion, so people can vote for it via ⬆️ button at above post.

Isn't all that needs to be done to install this:
```sh
pip3 install ccxt
```
as of https://pypi.org/project/ccxt/?
